structs/config.go

```diff
@@ -31,6 +31,7 @@ type PageContext struct {
 	MainFacebook    string `toml:"mainFacebook"`
 	NewsTwitter     string `toml:"newsTwitter"`
 	MyRadioAPIKey   string `toml:"myRadioAPIKey"`
+	ODName          string `toml:"odName"`
```

Comment:
have a configurable name for the on demand stuff, but then hardcode the url anyway. Probably not worth bothering with this config var

Reply:
Hmm, I guess, but it's here now, may as well keep it.